cortexlab/+git/update.m

```diff
@@ -20,11 +20,11 @@ function update(fatalOnError, scheduled)
 % Get the path to the Git exe
 gitexepath = getOr(dat.paths, 'gitExe');
 if isempty(gitexepath)
-  [~,gitexepath] = system('where git');
+  [~,gitexepath] = system('where git'); % this doesn't always work
 end
 gitexepath = ['"', strtrim(gitexepath), '"'];
 
-% Temporarily change directory into Rigbox
+% Temporarily change directory into Rigbox to git pull
 origDir = pwd;
 cd(root)
 
```

Comment:
why is the "check for changes" code below commented out?

Reply:
Because it didn't work the way I had hoped, and running git pull when there are no changes doesn't lose you much extra time.